lib/spack/docs/configuration.rst

```diff
@@ -202,12 +202,12 @@ supply ``-p`` to Spack on the command line, before any subcommands.
 .. _spack-p:
 
 ^^^^^^^^^^^^^^^^^^^
-``spack --profile``
+``spack -p``
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
I agree.  Currently all `conf.py` does is `glob` for sections whose labels start with `spack-`.  I think it might better to make that explicit... e.g. require a `cmd-` prefix or similar.